views/test.ejs

```diff
@@ -36,6 +46,48 @@
 
         $scope.changelocale = function() {
           $scope.request.request.locale = $scope.locale;
+          $scope.request.context.System.apiEndpoint = $scope.apiEndpoint;
+
+          if ($scope.locale === 'en-US') {
+            $scope.request.context.System.apiEndpoint = 'https://api.amazonalexa.com/';
+          } else {
+            $scope.request.context.System.apiEndpoint = 'https://api.eu.amazonalexa.com';
```

Comment:
I think we should implement specific regions specifically and default to `https://api.amazonalexa.com` for others.